### PR TITLE
Create Style manual for contributors

### PR DESCRIPTION
**Pull request by Soitora**
Hoooh this was messy to make, when I was asked to do this I did not want to make it :S
Hopefully it's to some use to people when creating faq entries.

This includes dependency updates too as well as adjusting *very slightly* some content that is mentioned in the style guide.

Also includes two experimental accordions for bottom of FAQ.

### CODE REVIEW

File: src/sandbox/style-guide.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+- **DO NOT** forget to at least check for basic grammar issues, this includes not having a capital letter in your first word and forgetting commas/periods.
+
+- **DO** preferably try to build the enviroment and test it locally first before submitting your PR, ask for help with this if you need it.
+
+- **DO** use guide containers for quickly showing how to achieve something:
+> ::: guide Instructions
+> Enable it by going to <Navigation item="library"/> → <Navigation item="filter"/> → **Display** section and then checking the **Download badges** option at the bottom.
+> :::
+
+- **DO** include video guides as a text under the header:
```
Comment (GHOSTBEAR) [inline src/sandbox/style-guide.md:34]: Should there be any guidelines on how videos to make videos? Like video format and that stuff so there is some sort of consistency between videos.

File: src/sandbox/style-guide.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+- **DO NOT** be afraid to ask for help, this includes clarifying what some parts on this page mean.
+
+- **DO** write each segment as a question:
+> #### How can I see how many chapters I've downloaded?
+
+- **DO** write your instructions as clear as possible, preferably check with other people first and see if they can help you improve your language used.
+
+- **DO NOT** write noticably sloppily, this makes it harder for both us and the user to understand.
+
+- **DO NOT** forget to at least check for basic grammar issues, this includes not having a capital letter in your first word and forgetting commas/periods.
```
Comment (GHOSTBEAR) [inline src/sandbox/style-guide.md:25]: [Grammarly ](https://www.grammarly.com/) and [Hemingway app](http://www.hemingwayapp.com/) are tools that can be used to help with grammar and clearness. If you want to give them some direction on what they can use to check for grammar issues and that kind of things

File: src/sandbox/style-guide.md (hunk 1 above)
Comment (Soitora) [reply src/sandbox/style-guide.md:34]: I'll add something about this soon, good idea

File: src/sandbox/style-guide.md (hunk 2 above)
Comment (Soitora) [reply src/sandbox/style-guide.md:25]: Sure I can add these resources

**Review by GHOSTBEAR — APPROVED** [review]

**Pull request merged**